pkg/plugins/trivy/plugin_test.go

```diff
@@ -7629,3 +7629,160 @@ func TestExcludeImages(t *testing.T) {
 		})
 	}
 }
+
+func TestParseImageRef(t *testing.T) {
+	testCases := []struct {
+		name string
+		// args:
+		imageRef string
+		imageID  string
+		// result:
+		registry v1alpha1.Registry
+		artifact v1alpha1.Artifact
+		err      error
+	}{
+		{
+			name:     "1. short image ref with latest tag",
```

Comment:
you can eliminate numbering in tests
```suggestion
			name:     "short image ref with latest tag",
```